server/config/releases.exs

```diff
@@ -55,13 +55,6 @@ config :realtime,
   jwt_secret: jwt_secret,
   jwt_claim_validators: jwt_claim_validators
 
-secret_key_base =
-  System.get_env("SECRET_KEY_BASE") ||
-    raise """
-    environment variable SECRET_KEY_BASE is missing.
-    You can generate one by calling: mix phx.gen.secret
-    """
-
 config :realtime, RealtimeWeb.Endpoint,
   http: [:inet6, port: app_port],
-  secret_key_base: secret_key_base
+  secret_key_base: jwt_secret
```

Comment:
@soedirgo 

> which we don't really use anyway

I think you're right. I don't think Realtime server is using it to sign/verify anything. 

Perhaps we should consider removing it altogether to prevent future confusion?

For folks who are setting `SECURE_CHANNELS` to `false`, `jwt_secret` isn't mandatory. If user doesn't input `jwt_secret`, then `secret_key_base` just defaults to a meaningless empty string.

Reply:
Yeah, probably better to just remove it. cc @kiwicopple

Reply:
agreed!